src/utils/wallet.js

```diff
@@ -254,12 +250,20 @@ export class Wallet {
     }
 
     async addAccessKey(accountId, contractId, publicKey) {
-        return await this.getAccount(accountId).addKey(
+        const res = await this.getAccount(accountId).addKey(
             publicKey,
             contractId,
             '', // methodName
             ACCESS_KEY_FUNDING_AMOUNT
         )
+
+        await sendJson('POST', `${ACCOUNT_HELPER_URL}/account/seedPhraseAdded`, {
```

Comment:
this only needs to happen for `addAccessKeySeedPhrase` action, having it here would result in every app-specific access key to be registered as a seed phrase

Reply:
Great point! I've moved this logic to the action creator – I don't love the look of it, but maybe it's ok. Do you see a quick way to clean it up?

I have verified that it still works locally:

<img width="731" alt="Screen Shot 2020-04-07 at 13 02 33" src="https://user-images.githubusercontent.com/221614/78698427-51da1980-78d0-11ea-9fc2-e543a0e48916.png">

Reply:
@chadoh I think it makes sense to move more logic to actions. `wallet.js` seems like just historically ended up being almost "god object" kind of thing.

We probably just need to modularize actions better (for now way too much stuff ended up in `account.js`)